.github/workflows/build.yaml

```diff
@@ -206,12 +206,14 @@ jobs:
           path: ./dist/cache
           key: radius-coverage-${{ github.sha }}-${{ github.run_number }}
       - name: Login to GitHub Container Registry
+        if: github.ref == 'refs/heads/main'
         uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
       - uses: oras-project/setup-oras@v1
+        if: github.ref == 'refs/heads/main'
```

Comment:
What about here? https://github.com/radius-project/radius/pull/7258/files#diff-d0777657fa3fd81d23aaf7273e58aee453b04e67882517900c56daeef9b3e4c1R243

Does this need to change as well?

I noticed we're doing two different conditions.

- If "ref is main"
- If "repo is radius-project/radius"